mysql/mysqldump.go

```diff
@@ -201,14 +230,20 @@ func processCreateTable(conv *internal.Conv, stmt *ast.CreateTableStmt) {
 		if constraint.fk.Columns != nil {
 			fkeys = append(fkeys, constraint.fk)
 		}
+		if constraint.isUniqueKey {
```

Comment:
As discussed, we'd like to avoid doing spanner-specific transformations when transformation from source database to schema.go.

However, there is no simple way to do this right now. Let's keep as is. I've filed an issue to improve this part of HarbourBridge (see https://github.com/cloudspannerecosystem/harbourbridge/issues/105).

But let's add a TODO:

// TODO: Avoid Spanner-specific schema transformations in this file -- they should be
// only appear in toddl.go. This file should focus on generic transformation from source
// database schemas into schema.go.
//

Reply:
added TODO, and this can be resolved separately